CHANGELOG.md

```diff
@@ -436,17 +436,24 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### Parser
 
+#### Enhancements
+
+- The JSON parser now allows comments in files with the `.json` extension under the `.vscode` and `.zed` directories. Contributed by @Conaclos
```

Comment:
Since it's possible that users already created an override to support this, we should add a diff that shows that that override isn't needed anymore.

Also, I would love it if we could follow up with a PR to update the documentation.

Reply:
Done.